README.md

```diff
@@ -12,7 +12,7 @@ Best supported on Ubuntu 14.04 with default `gdb` or `gdb-multiarch` (e.g. with
 ## Installation
 
 1. Clone the repo: `git clone https://github.com/zachriggle/pwndbg`
-2. Add to `~/.gdbinit`: `source ~/pwndbg/gdbinit.py`
+2. Add to `~/.gdbinit`: `cd pwndbg && echo "source $PWD/gdbinit.py" >> ~/.gdbinit`
```

Comment:
Right. This just makes sure that step 1 wasn't run with an extra arg (different clone dir than pwndbg). I'll change.